src/sentry/static/sentry/app/views/settings/project/projectTeams.jsx

```diff
@@ -181,7 +182,7 @@ class ProjectTeams extends AsyncView {
       .map(team => ({
         value: team.id,
         searchKey: team.slug,
-        label: <TeamDropdownElement>#{team.slug}</TeamDropdownElement>,
+        label: <div className={styles.padding(2, 1)}>#{team.slug}</div>,
```

Comment:
I'd prefer a styled component for this, I think reading a component name is a win over a class names.

Reply:
ok. if that is the case I'll probably just bail on helpers completely. the only point of having them is to avoid making up names for functionless styled components. otherwise it's just extra code.

Instead i'm going to replace it with direct calls to scale, something like:

```css
margin: 0 ${scale(2)};
padding: ${scale(3)} ${scale(1)};
```